### PR TITLE
fix: hero section

### PR DESCRIPTION
**Pull request by Shurtu-gal**

### CODE REVIEW

**Comment by ashutosh-rath02** [comment]
image
Overflowing for mobile screens

File: src/components/Hero.jsx [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 import { HeroText2 } from './shared/typography';
 import { imgURLs } from '../data/eventsData';
 
 const Hero = () => {
 	return (
-		<div style={{ margin: '0 60px' }}>
-			<div className='flex flex-col md:flex-row'>
-				<div className='w-[100%] md:w-1/2 '>
-					<div className='h-1/3 flex flex-col justify-end ml-2' style={{ color: '#FF4409' }}>
+		<div className='mx-5 lg:mx-14 xl:max-w-[1483px]'>
```
Comment (ashutosh-rath02) [inline src/components/Hero.jsx:8]: Works for now but use container

**Review by ashutosh-rath02 — DISMISSED** [review]
LGTM🦒

File: src/components/Hero.jsx (hunk 1 above)
Comment (Shurtu-gal) [reply src/components/Hero.jsx:8]: container won't work it only sets the `max-width` of an element to match the `min-width` of the current breakpoint.

**Pull request merged**